test/extension.test.ts

```diff
@@ -1,8 +1,15 @@
 /* global suite, test */
 
-const vscode = require('vscode');
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
```

Comment:
In the linter it says there is no default export for vscode and assert.  The import I used takes everything that is on the module.exports object and imports it into a single variable.